docker/demo/README.md

```diff
@@ -0,0 +1,138 @@
+# Lumify Demo Docker Container
```

Comment:
Node.js, npm and grunt-cli are also required dependencies. We need to add documentation indicating either where to find them or how to install them.